Preferences/Preferences/Shortcuts.h

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
I think this definitions will be compiled into static area twice, for each include.
Need a better way to workaround this.
`extern` lead to unsatisfied dynamic linkage error

Reply:
You could also compile these constants into a static object file or library just so the strings are all sourced from the same binary instead of just the same source header. I'm not sure that's better or worse, either. Either way, they're going to be string included in every single source file that mentions the shortcuts, which shouldn't be too many, since it should only be the main application and the preferences pane.